.github/workflows/pull-upstream.yml

```
@@ -11,10 +11,10 @@ jobs:
     timeout-minutes: 15
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v3.3.0
 
     - name: Set up Python 3.7
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4.4.0
```

```suggestion
      uses: actions/setup-python@v4
```
Set only major and will always run the latest